assimilation_code/programs/obs_sequence_tool/obs_sequence_tool.f90

```diff
@@ -141,7 +155,7 @@ program obs_sequence_tool
          min_gps_height, edit_copy_metadata, new_copy_index,                 &
          edit_qc_metadata, new_qc_index, synonymous_copy_list,               &
          synonymous_qc_list, edit_copies, edit_qcs, new_copy_metadata,       &
-         new_qc_metadata, new_copy_data, new_qc_data, write_external_FOs
+         new_qc_metadata, new_copy_data, new_qc_data, external_FO_write_list
```

Comment:
The write_external_FOs variable was an all-or-nothing logical. I think this is more descriptive.
AFAIK, there is no namelist in the repo that actually had the 'write_external_FOs' variable specified.